src/legacy/core_plugins/interpreter/server/lib/route_expression/thread/index.js

```diff
@@ -36,6 +36,7 @@ export function getWorker() {
     const { type, value, id } = msg;
     if (type === 'run') {
       const { threadId } = msg;
+      if (!heap[threadId]) return;
```

Comment:
Why'd you add this? Is there a time when a "run" message would come in but the thread didn't exist? What was the condition?

Reply:
sorry, left over from some debuging ... yeah, i was hitting this, threadId being null, but it was due to some other issues i had in the code at the time.